package.json

```diff
@@ -113,11 +116,12 @@
     "react-use-websocket": "^2.9.1",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
-    "rollup-plugin-scss": "^2.6.1",
-    "sass": "~1.30.0",
+    "rollup-plugin-import-css": "^3.0.2",
```

Comment:
Is this actually used? I can't find it anywhere, specifically not in the rollup config file.

Reply:
Removed 👍